example/package.json

```diff
@@ -10,7 +10,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "@jellyfish-dev/react-native-membrane-webrtc": "https://github.com/jellyfish-dev/react-native-membrane-webrtc#jellyfish",
+    "@jellyfish-dev/react-native-membrane-webrtc": "https://github.com/jellyfish-dev/react-native-membrane-webrtc",
```

Comment:
this one is a little confusing, it should be installed automatically because it should be in dependencies of `react-native-client-sdk`, I haven't looked into why is it needed

Reply:
Without it builds on ios doesn't work :/